keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -749,6 +762,7 @@ Pick one of these and delete the rest.
 - [x] Feature gate (also fill in values in `kep.yaml`)
   - Feature gate name: ServerSideFieldValidation
   - Components depending on the feature gate: kube-apiserver
+  - Note: feature gate removed upon graduation to GA (1.27)
```

Comment:
```suggestion
  - Note: feature gate locked to true upon graduation to GA (1.27) and removed two releases later (1.29)
```

Reply:
https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation

> Rule #8: Feature gates must be deprecated when the corresponding feature they control transitions a lifecycle stage as follows. Feature gates must function for no less than:
>
> * Beta feature to GA: 6 months or 2 releases (whichever is longer)
> ...

we lock to true and give 2 releases for folks to update their invocations to drop explicitly setting the gate